.github/workflows/schemas.yaml

```diff
@@ -21,13 +21,16 @@ jobs:
   generate:
     name: Generate
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby: [2.7, 3.1]
```

Comment:
The changes in this file aren't necessary as it just generates the schemas and ensures they match the ones in the commit history.